UNRELEASED.md

```diff
@@ -2,6 +2,10 @@
 
 > Place your changes below this line.
 
+**Added:**
+- Backpack/Icon
+  - New `flight-landing`, `flight-takeoff` and `aircraft` icons.
```

Comment:
Can we use the `Objective-C` names instead?

Reply:
Are they these: `BPKIconNameFlightLanding` or `@"flight-landing"`? I used a previous PR as reference

Reply:
Done 😄